README.md

```diff
@@ -104,7 +104,13 @@ yarn coverage
 
 ### Setup
 
-Configure the `.env` variables.
+```bash
+cast wallet import MAINNET_DEPLOYER --interactive
```

Comment:
What do you think about giving devs the choice to name the wallet the way they want? We could do it through the `.env` file.

Reply:
hmmm seems unnecessary to build an .env to name the deployer, isn't foundry storing it locally? 
is "forcing a dev to choose a name" (setting the env), vs "just using `MAINNET_DEPLOYER`" and allowing the dev to change the package.json

Reply:
Changing the `package.json` would require every dev to declare (or redeclare) such account locally in order to be able to run `yarn deploy`. However, I agree with that forcing a dev to choose a name is generally undesired. Maybe we could update `MAINNET_DEPLOYER` at the very beginning of each project, setting it to a more ad hoc account name.